src/db/migrate.rs

```diff
@@ -618,6 +618,88 @@ pub fn migrate(version: Option<Version>, conn: &mut Client) -> CratesfyiResult<(
             DROP INDEX github_repos_stars_idx;
             ",
         ),
+        migration!(
+            context,
+            27,
+            // description
+            "Add gitlab handling: creation of the new repositories table which replaces and extend \
+             github_repos",
+            // upgrade query
+            "
+                CREATE TABLE repositories (
+                    id SERIAL PRIMARY KEY,
+                    host VARCHAR NOT NULL,
```

Comment:
I would be tempted to make this a `<forge type>:<host>` pair, i.e. `github:github.com`, `gitlab:gitlab.com`, `gitlab:gitlab.freedesktop.org`, so that even without the list of hostnames in the source code you can identify what sort of forge it is.

Reply:
But that would require to extract the host for gitlab. Not sure if it's a good idea...

Reply:
I agree with this approach, especially because otherwise we wouldn't be able to distinguish between a GitLab instance and another self-hosted forge, if we're going to add support for it in the future.